browser-test/src/support/setup-jest.ts

```diff
@@ -0,0 +1,37 @@
+import axe = require('axe-core')
+
+interface CustomMatchers<R = unknown> {
+  toHaveNoA11yViolations(): R
+}
+
+declare global {
+  namespace jest {
+    interface Expect extends CustomMatchers {}
+    interface Matchers<R> extends CustomMatchers<R> {}
+    interface InverseAsymmetricMatchers extends CustomMatchers {}
+  }
+}
+
+// Custom matcher that outputs accessibility violations using axe.
+expect.extend({
+  toHaveNoA11yViolations(results: axe.AxeResults) {
+    const numViolations = results.violations.length
+    const pass = numViolations == 0
+    if (pass) {
+      return {
+        message: () => `Expected axe accessibility violations, found none`,
```

Comment:
Not blocking: Nit: Since there's no string interpolation, regular quote characters can be used. The backtick signifies that it's a template string. The effect at runtime will be the same, but when I saw a backtick, I was looking for the interpolated variables by reflex.

Reply:
Thanks for the context! Switched to single quote.